bin/runtime/src/lib.rs

```diff
@@ -449,21 +445,24 @@ impl_runtime_apis! {
         ) -> sp_inherents::CheckInherentsResult {
             data.check_extrinsics(&block)
         }
+
+        fn random_seed() -> <Block as BlockT>::Hash {
+            RandomnessCollectiveFlip::random_seed()
```

Comment:
I'm pretty sure we were supposed to remove the flip pallet.

Reply:
It turns out that the trait `sp_block_builder::BlockBuilder<Block>` has a method `fn random_seed() -> <Block as BlockT>::Hash` where the flip pallet is used. It is removed in newer versions.

Reply:
Ah, interesting. How sure are you nothing uses this method? If pretty sure then I'd prefer the `default` implementation, without the additional pallet.